### PR TITLE
fix: some typos and syncPod outdated workflow annotation

### PR DESCRIPTION
**Pull request by qmloong**
#### What type of PR is this?
fix some typos 

#### What this PR does / why we need it:
fix typo and update syncPod workflow annotation

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by matthyx** [comment]
/kind cleanup
/release-note-none
not sure we're really interested in these kind of PRs, wdyt @ehashman ?

**Comment by qmloong** [comment]
> /assign @sjenning
> 
> Please help review this pr when you free? reviewers look very busy. Just a minor change. If the change doesn't make any sense, I will close it. : )

@vishh @mtaufen  hey, master. Please take time to help review this pr.

File: pkg/kubelet/kubelet.go [hunk 1]
```diff
@@ -1742,7 +1743,7 @@ func (kl *Kubelet) syncPod(ctx context.Context, updateType kubetypes.SyncPodType
 		for _, r := range result.SyncResults {
 			if r.Error != kubecontainer.ErrCrashLoopBackOff && r.Error != images.ErrImagePullBackOff {
 				// Do not record an event here, as we keep all event logging for sync pod failures
-				// local to container runtime so we get better errors
```
Comment (haosdent) [inline pkg/kubelet/kubelet.go:1745]: Add period at the end to make it become a complete sentence.
`local to container runtime, so we get better errors.`

Comment (qmloong) [reply pkg/kubelet/kubelet.go:1745]: ok

**Comment by dims** [comment]
/retest
/assign @derekwaynecarr 

NOTE: 2 lines updated, both are comments so should be quick. Can you please take a look?

**Review by mrunalp — APPROVED** [review]

**Pull request merged**